### PR TITLE
Supervisor updates

### PR DESCRIPTION
**Pull request by jlashner**
Some changes to HWP control logic from testing on SATP3 hardware with kyohei.

## Description

A few changes to supervisor control logic including:
- addition of PCU control into various control states
- nailing down braking logic

## How Has This Been Tested?

Tested on SATp3 hardware

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: socs/agents/hwp_supervisor/agent.py [hunk 1]
```diff
@@ -626,27 +644,88 @@ def update(self, clients, hwp_state):
...
+                        freq_tol=state.freq_tol,
+                        freq_tol_duration=state.freq_tol_duration,
+                        direction=state.direction,
+                    ))
+                    return
+
+                if time.time() - state.start_time < state.check_wait_time:
+                    return
 
+                time.sleep(5)
```
Comment (BrianJKoopman) [inline socs/agents/hwp_supervisor/agent.py:685]: Why the 5 second sleep after waiting for the `check_wait_time` duration? Should `check_wait_time` just be 5 seconds longer?

File: socs/agents/hwp_supervisor/agent.py [hunk 2]
```diff
@@ -1086,6 +1139,7 @@ def set_const_voltage(self, session, params):
 
     @ocs_agent.param('freq_tol', type=float, default=0.05)
     @ocs_agent.param('freq_tol_duration', type=float, default=10)
+    @ocs_agent.param('brake_voltage', type=float, default=10.)
```
Comment (BrianJKoopman) [inline socs/agents/hwp_supervisor/agent.py:1142]: Add to method signature override in first line of docstring.

**Review by BrianJKoopman — COMMENTED** [review]
Looks good to me, one docstring update to make, and a question about one of the `time.sleep` calls.

File: socs/agents/hwp_supervisor/agent.py (hunk 1 above)
Comment (jlashner) [reply socs/agents/hwp_supervisor/agent.py:685]: yes.... that's a good point

**Review by BrianJKoopman — APPROVED** [review]
Thanks for the updates!

**Pull request merged**